extensions/amp-video-service/amp-video-service.md

```diff
@@ -0,0 +1,42 @@
+<!--
```

Comment:
I don't think this needs a public `.md` file, would be confusing for it to show up on ampproject.org as it is not user facing. I suggest moving any important documentation to the top of the `.js` file

Reply:
Good call, done.